src/components/AudioPlayer/AudioPlayer.tsx

```diff
@@ -116,69 +119,69 @@ class AudioPlayer extends React.Component<AudioPlayerProps> {
           <source src={src} />
         </audio>
         <Grid className={css(classes['player-container'], player)} elevation={elevation} rounded={rounded} component={Paper} alignContent="center" justify="center" alignItems="center" spacing={8} container>
-            <Grid md={1} item>
-              <LoopStatusIcon
-                className={css(classes['player-icon-disabled'], loopIcon, {
-                  [classes['player-default-icon']]: isLoopEnable,
-                })}
-                onClick={() => this.triggerAction(Player.Status.LOOP)}
-                focusable="true"
-              />
-            </Grid>
-            <Grid md={1} item>
-              <PlayStatusIcon
-                className={css(
-                  classes['player-default-icon'],
-                  classes['player-main-icon'],
-                  playIcon
-                )}
-                onClick={() => this.triggerAction(Player.Status.PLAY)}
-                focusable="true"
-              />
-            </Grid>
-            <Grid md={1} item>
-              <MuteStatusIcon
-                className={css(classes['player-icon-disabled'], muteIcon, {
-                  [classes['player-default-icon']]: isMuteEnable,
-                })}
-                onClick={() => this.triggerAction(Player.Status.MUTE)}
-                focusable="true"
-              />
-            </Grid>
-            <Grid md={2} item>
-                <Typography
-                    className={css(classes['player-text'], text)}
-                    align="center"
-                    noWrap
-                >
-                    {getFormattedTime(current)}
-                </Typography>
-            </Grid>
-            <Grid md={5} item>
-                <Slider
-                    onChange={(_, progress) =>
-                      this.handleChange(progress, this.player)
-                    }
-                    classes={{
-                      root: css(classes['player-slider-container'], slider),
-                      track: css(classes['player-slider-track'], track),
-                      thumb: css(classes['player-slider-thumb'], thumb),
-                    }}
-                    variant="determinate"
-                    color="secondary"
-                    value={progress}
-                />
-            </Grid>
-            <Grid md={2} item>
-                <Typography
-                    className={css(classes['player-text'], text)}
-                    align="center"
-                    noWrap
-                >
-                    {getFormattedTime(duration)}
-                </Typography>
-            </Grid>
+          {showLoopIcon && <Grid md={1} item>
```

Comment:
Hi! Would you mind to wrap the `Grid` with parenthesis?

Reply:
@JonatanSalas Hi! To wrap like `{showLoopIcon  && (<Grid>...</Grid>)}`?

Reply:
Yep! Anyway, there's a `.prettierrc.js` file that should handle this case 😅. Do you use `vscode` with `prettier` extension?

Reply:
Let's resolve this, and I will review it in my local machine.

Reply:
@JonatanSalas Yeah, I use `vscode` but don't use `prettier` extension. I will install it. Thank you!